frechet_audio_distance/fad.py

```diff
@@ -309,8 +360,8 @@ def update(*a):
         for fname in os.listdir(dir):
             res = pool.apply_async(
                 load_audio_task,
-                args=(os.path.join(dir, fname), self.sample_rate, dtype),
-                callback=update
+                args=(os.path.join(dir, fname), self.sample_rate, self.channels, dtype),

```

Comment:
I suspect `self.channels` is not defined here, but please correct me if I am wrong.

Reply:
yep, it looks like I erased the definition when commiting